### PR TITLE
Blueshield Officer and Nanotrasen Representative changes.

### PR DESCRIPTION
**Pull request by Verbalase**
## About the PR
Removed all access from the Nanotrasen Representative and Blueshield Officer and changed the playtimes to better fit what the Paradise Station wiki says. There's also 

## Why / Balance
According to Paradise Station, neither of the roles even have AA. The jobs do not require the accesses either and are fine with less of them.

The playtimes according to the Paradise Station wiki require 50 hours of command playtime so... Yeah. The Blueshield Officer I don't believe should require just command access playtime since you are also acting as a bodyguard and will have to do more. Do tell if otherwise though.

One last thing was also changing who the Blueshield answers to since it IS said that the Blueshield Officer answers to Central Command and the Nanotrasen Representative.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Changed the Blueshield and NTR's accesses, who the Blueshield answers to, and the playtime requirements for both jobs.

### CODE REVIEW

**Comment by Aidenkrz** [comment]
115 hours is a bit extreme considering the playtime requirements for other command roles.

**Comment by Verbalase** [comment]
> 115 hours is a bit extreme considering the playtime requirements for other command roles.

What should I change then for it?

**Comment by Aidenkrz** [comment]
25 sec 25 command? total of 65 hours?

**Comment by Verbalase** [comment]
> 25 sec 25 command? total of 65 hours?

Done. Anything else?

**Review by Piras314 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml [hunk 1]
```diff
@@ -6,22 +6,30 @@
...
       department: Security
-      time: 54000 # 15 hours
+      time: 90000 # 25 hours
   weight: 20
   startingGear: BlueshieldOfficerGear
   icon: "JobIconNanotrasen"
   requireAdminNotify: true
   joinNotifyCrew: true
-  supervisors: job-supervisors-centcom
+  supervisors: "the Nanotrasen Representative and Central Command"
```
[the remark this answers is not in the document]
Comment (Verbalase) [reply Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml:21]: But that would conflict with what the Nanotrasen Representatie would get. The Blueshield answers to the NTR and CC.

[the remark this answers is not in the document]
Comment (Piras314) [reply Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml:21]: make a new localisation job-supervisors-ntr-centcom then

**Comment by Aidenkrz** [comment]
also stop making fucking branches on the main repo, fork it.

**Comment by Verbalase** [comment]
> also stop making fucking branches on the main repo, fork it.

Yessir yessir

File: Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Verbalase) [reply Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml:21]: > make a new localisation job-supervisors-ntr-centcom then, mb

am i fine to put this in the job folder in en-us or do i have to put it in the goobstation folder still?

File: Resources/Locale/en-US/Goobstation/job/department-desc.ftl
Comment (Piras314) [inline Resources/Locale/en-US/Goobstation/job/department-desc.ftl:1]: why does this exist

File: Resources/Locale/en-US/Goobstation/job/department.ftl
Comment (Piras314) [inline Resources/Locale/en-US/Goobstation/job/department.ftl:1]: why

File: Resources/Locale/en-US/Goobstation/job/job-description.ftl
Comment (Piras314) [inline Resources/Locale/en-US/Goobstation/job/job-description.ftl:1]: why

File: Resources/Locale/en-US/Goobstation/job/job-names.ftl
Comment (Piras314) [inline Resources/Locale/en-US/Goobstation/job/job-names.ftl:1]: why

File: Resources/Locale/en-US/Goobstation/job/job-supervisors.ftl
Comment (Piras314) [inline Resources/Locale/en-US/Goobstation/job/job-supervisors.ftl:1]: why

File: Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Piras314) [reply Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml:21]: goobstation folder, but why are you copying all this wizden stuff over into goobstation files/folder.... don't do that.

**Review by Piras314 — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/Goobstation/job/department-desc.ftl
Comment (Verbalase) [reply Resources/Locale/en-US/Goobstation/job/department-desc.ftl:1]: was deleting

File: Resources/Locale/en-US/Goobstation/job/department.ftl
Comment (Verbalase) [reply Resources/Locale/en-US/Goobstation/job/department.ftl:1]: was deleting

File: Resources/Locale/en-US/Goobstation/job/job-supervisors.ftl
Comment (Verbalase) [reply Resources/Locale/en-US/Goobstation/job/job-supervisors.ftl:1]: was deleting

File: Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Verbalase) [reply Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml:21]: so you want this in the goob folder but how specifically..?

[the remark this answers is not in the document]
Comment (Verbalase) [reply Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml:21]: like do i make a job-supervisors.ftl file in the goob folder then put job-supervisors-ntr-centcom?

[the remark this answers is not in the document]
Comment (Piras314) [reply Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml:21]: so make job-supervisors.ftl in a goob folder then put job-supervisors-ntr-centcom = the Nanotrasen Representative and Central Command blablabla and change where it says in this file "the Nanotrasen blablabla" to job-supervisors-ntr-centcom

Comment (Piras314) [inline Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml:21]: final step, change this line to supervisors: job-supervisors-ntr-centcom

**Review by Piras314 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml (hunk 1 above)
Comment (Verbalase) [reply Resources/Prototypes/Goobstation/Roles/Jobs/Command/blueshield_officer.yml]: Is this all it and good or did I confuse another thing. Accidentally mistaked ntr and blueshield together at first.

**Comment by Verbalase** [comment]
Tested and it seems to work just fine.

**Review by Piras314 — APPROVED** [review]

**Comment by Verbalase** [comment]
im now a master coder

durk does this mean i can self-merge now

**Comment by Piras314** [comment]
no.

**Pull request merged**